packages/core/middleware/__tests__/test-data/getAgnosticStatusCode.ts

```diff
@@ -0,0 +1 @@
: {jwwjigepu: {cvakgr: {qxazk: {jvcldlqno: {oekno: {vvftsltgi: {trkhloufc: {kuylwn: {anhazxhq: 'wehnkc'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, jeudg: {tchcfax: {xbwuj: {depkqsl: {fgadqrq: {jnyrihilh: {uhudl: {deeznoyau: {cfnlzvs: {gpdlp: {ytigzwr: {vvjwz: {psfrcpfjd: {zjuyixwft: {brkthq: {vvepqrvr: {jqstetn: {oinvblhv: {nwvvht: {qhuijkn: {ptdye: {oyobhx: {iqszfgp: {evenqn: {cqhqwq: {wanpbuqi: {olajv: {ojnzbki: {hclnnf: {qfxosdgln: {vtbfoyhb: 'jjvylwk'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, pkpzkzuco: {ilxkrzpey: {nevffcblv: {kyrpiit: {pihnown: {bhdvzvch: {fahlejj: {fqqiv: {rksork: {ncphyqfvg: {ngnrnztj: {dhaeq: {lbchhwrhh: {ibtybf: {kgehun: {ayhvgtd: {cryircu: {dbpilcpee: {euzqg: {ktgew: {anocqxvv: {wxvsn: {xpqqjv: {ktizg: {ojjxheff: {kqlvdt: {lgwwbsu: {cprsxgwde: {clhvv: {hflwgf: {uklleoq: 'aldfu'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, dpacg: {qepvfqxs: {cwtpart: {ksgjnti: {cthed: {dgedkk: {txceg: {wazysuc: {awwgddf: {wlfducdee: {sbnshwsn: {uhijcqko: {lpzwqveo: {lornkd: {nekpwbwlv: {zrnkpc: {jwcsg: {hzufclv: {csobtfr: {jsvfkre: {fblhjhlrl: {iohvfvgu: {lthjvalys: {qteku: {puipvzf: {ikpxko: {pztyozcz: {jftrrpdz: {vyvqdok: {efsnegzp: {xtdrp: 'jjiqsyh'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, hjdhbv: {xlcct: {uuffju: {dwepnqzi: {exfthqlz: {kbjalv: {uunwbxqk: {nctybwojf: {pwhnklut: {hcoeq: {yocacp: {fvjkdd: {lxbrzdekl: {kecsg: {dypihzru: {oqicfcts: {dfesdv: {czrlcen: {oznhoo: {brzihbpsl: {xakswett: {yldlzog: {ukfbls: {vbxvynapv: {ovdsepfds: {nsnqauw: {wtqssah: {kpqdygjxq: {bnpjgnw: {ybxcu: {ohnfat: 'qybbni'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, nudcpeqr: {fwfhjuuv: {znbsb: {pfala: {dwqqkniq: {hyogbgvi: {uodas: {oafhvkjb: {gbdvx: {rxvgi: {jtrubcynk: {ujkdqzbvv: {ghluopvwy: {ldbynhzx: {bizoarj: {twznphw: {cfxidk: {eksdtdynh: {wkffuqld: {ipdena: {qyhratrdn: {uzvzil: {ijgyzwl: {ptcph: {iynxig: {vijfog: {tboaj: {hrrqyuled: {ofiypaq: {uttwecpr: {xxuahiah: 'zehsflg'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, wbrqi: {njudperci: {hfpulb: {foblnkkv: {hzkxn: {gwoyopcj: {yfbhzsk: {qfknl: {rlqkpzqzu: {tdvrapw: {ocivzod: {rynubkxan: {ruqea: {scbwnpjsr: {eaotr: {tkualiveo: {bogbpg: {qakadbzxw: {pczkx: {apicesut: {rsbwh: {kwyfwfza: {cuusz: {deskjjje: {xxzdtpwp: {zvtiuwx: {wphlurw: {udzjej: {byhbkji: {belpcq: {tgvuwofe: 'srhcwozvs'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, bqeglxii: {oiotjjsbv: {rcvjuuhrg: {qjckugu: {zvkdevzsl: {nuafosfej: {enbol: {hgajrtwdo: {laohfy: {udfut: {ylrgl: {vpznjjuj: {divlywfdy: {rpkeq: {pxknrsh: {lvppq: {vrbowdoz: {iqgbkog: {zapqcy: {fqhhjw: {dzzsnwu: {pekzx: {pnldqbtbt: {yjbjp: {oqiigtlsh: {qjtvi: {ejpspcid: {udpllip: {bdfftfnqv: {khrbrwtg: {lgtddknt: 'erklu'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, djcursvu: {tsxsk: {zoxkhiqh: {tcnfvrrju: {oknzclj: {wjywr: {nizqsdis: {couchwztf: {uyhibb: {jovsxixrs: {dsnpe: {hxkwrcyb: {hftpqeqxx: {fzcjn: {jgqkjn: {bljbgehd: {uczzjzqe: {agfpfu: {uvrhkp: {cduipvn: {uabpit: {aufuo: {gttfnecq: {fypdb: {lucceiy: {skpljtq: {hagguj: {hsjyqho: {aufrz: {tqaei: {jjhhn: 'jalvos'}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}, cdzobg: {dqxexaqq: {rncbknp: {dqyiao: {pikjen: {ohkyucjbp: {twwwyr: {xqblfsgg:
```

Comment:
Can't this mock be generated on demand??

Reply:
I wanted to do it on first place. I've decided not to do it, to not to waste time during tests on generating dynamic object and not to increase load of processor. For really big objects it can take a while to generate it.